performance/benchmarks/bm_regex_dna.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+"""
+The Computer Language Benchmarks Game
+http://benchmarksgame.alioth.debian.org/
+
+regex-dna Python 3 #5 program:
+contributed by Dominique Wahli
+2to3
+modified by Justin Peel
+
+fasta Python 3 #3 program:
+modified by Ian Osgood
+modified again by Heinrich Acker
+modified by Justin Peel
+Modified by Christopher Sean Forgeron
+"""
+
+import bisect
+import re
+
+import perf
+from six.moves import xrange
+
+
+alu = (
+   'GGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGG'
+   'GAGGCCGAGGCGGGCGGATCACCTGAGGTCAGGAGTTCGAGA'
+   'CCAGCCTGGCCAACATGGTGAAACCCCGTCTCTACTAAAAAT'
+   'ACAAAAATTAGCCGGGCGTGGTGGCGCGCGCCTGTAATCCCA'
+   'GCTACTCGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCGGG'
+   'AGGCGGAGGTTGCAGTGAGCCGAGATCGCGCCACTGCACTCC'
+   'AGCCTGGGCGACAGAGCGAGACTCCGTCTCAAAAA')
+
+iub = list(zip('acgtBDHKMNRSVWY', [0.27, 0.12, 0.12, 0.27] + [0.02] * 11))
+
+homosapiens = [
+    ('a', 0.3029549426680),
+    ('c', 0.1979883004921),
+    ('g', 0.1975473066391),
+    ('t', 0.3015094502008),
+]
+
+
+def make_cumulative(table):
+    P = []
+    C = []
+    prob = 0.
+    for char, p in table:
+        prob += p
+        P += [prob]
+        C += [ord(char)]
+    return (P, C)
+
+
+def repeat_fasta(src, n, nprint):
+    width = 60
+
+    is_trailing_line = False
+    count_modifier = 0.0
+
+    len_of_src = len(src)
+    ss = src + src + src[:n % len_of_src]
+    # CSF - It's faster to work with a bytearray than a string
+    s = bytearray(ss, encoding='utf8')
+
+    if n % width:
+        # We don't end on a 60 char wide line
+        is_trailing_line = True
+        count_modifier = 1.0
+
+    # CSF - Here we are stuck with using an int instead of a float for the loop,
+    # but testing showed it still to be faster than a for loop
+    count = 0
+    end = (n / float(width)) - count_modifier
+    while count < end:
+        i = count*60 % len_of_src
+        nprint(s[i:i+60] + b'\n')
+        count += 1
+    if is_trailing_line:
+        nprint(s[-(n % width):] + b'\n')
+
+
+def random_fasta(table, n, seed, nprint):
+    width = 60
+    r = range(width)
+    bb = bisect.bisect
+
+    # If we don't have a multiple of the width, then we will have a trailing
+    # line, which needs a slightly different approach
+    is_trailing_line = False
+    count_modifier = 0.0
+
+    line = bytearray(width + 1)    # Width of 60 + 1 for the \n char
+
+    probs, chars = make_cumulative(table)
+
+    # pRNG Vars
+    im = 139968.0
+    #seed = 42.0
```

Comment:
The variable is still used. Do you suggest to replace the variable by its value in the code?